app/mailers/twilio_sender.rb

```diff
@@ -2,23 +2,91 @@
 
 # TwilioSender: Methods to interact with Twilio REST API
 class TwilioSender
+  TWILIO_ERROR_CODES = {
+    invalid_to_number: '21211',
+    blocked_number: '21610',
+    invalid_number: '21614',
+    unsupported_region: '21408',
+    unreachable_unavailable: '30003',
+    unavailable_ineligible: '30004',
+    non_existent_or_off: '30005',
+    sms_ineligible: '30006',
+    carrier_filter: '30007',
+    unknown_error: '30008'
+  }.freeze
```

Comment:
How about including the error message strings in this hash as well since they're all just static strings. Something like
```ruby
TWILIO_ERRORS = {
  invalid_to_number: { code: '21211', message: 'Invalid recipient phone number.' },
  blocked_number: { code: '21610', message: 'Recipient phone number blocked communication with Sara Alert' },
  # etc..
}
```
and then the `handle_twilio_error_codes` method below becomes much simpler:
```ruby
def self.handle_twilio_error_codes(patient, error_code)
  BlockedNumber.find_or_create_by(phone_number: monitoree_number) if error_code == TWILIO_ERROR_CODES[:blocked_number][:code]
  err_msg = find { |k, v| v[:code] == error_code }&.second&[:message] || 'An unknown error has been encountered by the messaging system.'
  dispatch_errored_contact_history_items(patient, error_message)
end
```